src/deploymentModals.jsx

```diff
@@ -22,8 +22,8 @@ import React, { useState } from 'react';
 import { Alert } from "@patternfly/react-core/dist/esm/components/Alert";
 import { Button } from "@patternfly/react-core/dist/esm/components/Button";
 import { Checkbox } from '@patternfly/react-core/dist/esm/components/Checkbox';
-import { Modal } from "@patternfly/react-core/dist/esm/components/Modal";
-import { Text } from "@patternfly/react-core/dist/esm/components/Text";
+import { Content } from "@patternfly/react-core/dist/esm/components/Content";
+import { Modal } from '@patternfly/react-core/dist/esm/deprecated/components/Modal';
```

Comment:
:eyes: modals are deprecated? Or there's a new one with different API?

Reply:
Yes, there is a new modal in pf6 that supports fewer props and there are new sub-components `ModalHeader`, `ModalBody`, `ModalFooter`